javascript/javascript2/week1/week1Js2.js

```diff
@@ -0,0 +1,107 @@
+
+// shortest Word;
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function shorterword(danishWords) {
+    let shortestWord;
+    for (let i = 0; i < danishWords.length; i++) {
+        if (danishWords[i].length < danishWords[i++].length) {
+            shortestWord = danishWords[i];
+        }
+        else {
+            shortestWord = danishWords[i++]
+        }
+    }
+    return shortestWord;
+} 
+
+
+console.log(shorterword(danishWords))  // returns 'ø'
```

Comment:
Code looks fine except a little changes need to be done.
1. seems this function returning different word in different runs.  seems there might need to make little changes to for loop. 
reference: function shorterword(danishWords) {
    let shortestWord = danishWords[0];                   //little change here
    for (let i = 0; i < danishWords.length; i++) {
        if (danishWords[i].length < shortestWord.length) {.  // small change here
            shortestWord = danishWords[i]; 
        }
    }
  
    return shortestWord;
} 

//no need of else statement.
You can also do it many other ways , this is just for your reference.